ignite/pkg/cosmosaccount/cosmosaccount.go

```diff
@@ -240,12 +259,21 @@ func (r Registry) ExportHex(name, passphrase string) (hex string, err error) {
 		return "", err
 	}
 
-	return keyring.NewUnsafe(r.Keyring).UnsafeExportPrivKeyHex(name)
+	return unsafeExportPrivKeyHex(r.Keyring, name, passphrase)
+}
+
+func unsafeExportPrivKeyHex(kr keyring.Keyring, uid, passphrase string) (privkey string, err error) {
```

Comment:
@ilgooz is it still unsafe since the method invoked on `kr` no longer has the `Unsafe` prefix ?

Reply:
This method mimics how the previous `kr` method worked, so I opted to keep the name the same.

Reply:
Also: it's taken from [here](https://github.com/cosmos/cosmos-sdk/blob/b2b29d4909395ce3bf6944ce0b740fb5a689b218/client/keys/export.go#L98)

Reply:
ok thx!